internal/ingress/status/status.go

```diff
@@ -145,7 +145,7 @@ func (s statusSync) Run() {
 			cancelContext = newLeaderCtx(ctx)
 		},
 		OnNewLeader: func(identity string) {
-			glog.Infof("new leader elected: %v", identity)
+			glog.Info("new leader elected: %v", identity)
```

Comment:
why are you changing this? it should be infof

Reply:
ok~~